src/assets/image.js

```diff
@@ -0,0 +1,13 @@
+import Logo from '../../public/logo.png';
```

Comment:
does public folder serve a purpose other than holding images?

Can we do something like:
-> src
  -> assets
    -> images > (*.png), images.js
    -> sass

Reply:
Will look into it